### DISCUSSION TITLE
[Console api-portal] How to retrieve Secret header?

### DISCUSSION THREAD

**Discussion by ghost** [mia-platform/community#321] [Q&A] [answered]
### Which section?

Console

### Which is the version?

11.2.0

### Description

Hi all!

How can I retrieve the secret that the api-portal of the Console is asking me to test the protected present APIs?

Thank you!

**Comment by davidebianchi** [comment]
Hi @ivanpasquariello! From the api key section inside the Console you can configure the api key: https://docs.mia-platform.eu/docs/development_suite/api-console/api-design/api-key

**Reply by ghost** [reply]
Thank you @davidebianchi ! Is this method valid also for the api-portal of the Console? In which project should I create the new Key?

**Reply by davidebianchi** [reply, marked as answer]
No, the console has static keys. We are removing the needs for the api key to simplify the API integration.
If you need an api key to contact Console, you should contact us and we give to you the api key to use in the API call.

**Reply by ghost** [reply]
Ok thank you @davidebianchi, I think that what are you mentioning is referring to the PaaS Console right? What about an on-premise installation? How can I retrieve/create this API Key? Thank you once again!

**Reply by davidebianchi** [reply]
It's the same flow also for the on premise installation.
